packages/scm/src/common/scm.ts

```diff
@@ -0,0 +1,124 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import URI from '@theia/core/lib/common/uri';
+import { Disposable, Event } from '@theia/core/lib/common';
+import { StatusBarEntry } from '@theia/core/lib/browser';
```

Comment:
common modules should not depend on browser modules, please review dependencies between new files that they don't violate project structuring conventions

Reply:
fixed